autogen.sh

```diff
@@ -9,7 +9,9 @@ autoreconf --install --warnings=all --force
 
 args=""
 if [ -f "$topdir/.config.args" ]; then
-	args="$args $(cat $topdir/.config.args)"
+	args="$(cat $topdir/.config.args)"
```

Comment:
I was wondering if we shouldn't allow passing some arbitrary files to `autogen.sh` with different parameter sets.

Something like:
```bash
config_args=$1
if test "x$1" = 'x'; then
    config_args="$topdir/.config.args"
fi
args=""
if [ -f "$config_args" ]; then
...
fi
```